tests/2016-03-01/Microsoft.Insights.alertrules.tests.json

```diff
@@ -1,99 +1,99 @@
-            {
-              "$type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleWebhookAction, Microsoft.WindowsAzure.Management.Mon.Client",
-              "odata.type": "Microsoft.Azure.Management.Insights.Models.RuleWebhookAction",
-              "serviceUri": "https://localhost/foo"
-            }
-          ]
-        }
-      }
-    },
-    {
-      "name": "Test Invalid Alert - Missing required property",
-      "expectedErrors": [
-        {
-          "message": "Missing required property: properties",
-          "dataPath": "/",
-          "schemaPath": "/required/3"
-        }
-      ],
-      "definition": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#resourceDefinitions/alertrules",
-      "json": {
-        "name": "Test Alert",
-        "type": "microsoft.insights/alertrules",
-        "apiVersion": "2016-03-01",
-        "location": "Central US"
-      }
-    },
-    {
-      "name": "Test Invalid alert rule - invalid name",
-      "expectedErrors": [
-        {
-          "message": "Invalid type: object (expected string)",
-          "dataPath": "/name",
-          "schemaPath": "/properties/name/type"
-        }
-      ],
-      "definition": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#resourceDefinitions/alertrules",
-      "json": {
-        "name": {},
-        "type": "microsoft.insights/alertrules",
-        "apiVersion": "2016-03-01",
-        "location": "Central US",
-        "properties": {
-          "name": "TestAlert",
-          "description": "Description of the alert, included in email",
-          "isEnabled": true,
-          "condition": {
-            "$type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.ThresholdRuleCondition, Microsoft.WindowsAzure.Management.Mon.Client",
-            "odata.type": "Microsoft.Azure.Management.Insights.Models.ThresholdRuleCondition",
-            "dataSource": {
-              "$type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleMetricDataSource, Microsoft.WindowsAzure.Management.Mon.Client",
-              "odata.type": "Microsoft.Azure.Management.Insights.Models.RuleMetricDataSource",
-              "resourceUri": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/montest/providers/Microsoft.Compute/virtualMachines/Tailfeather",
-              "metricName": "\\Memory\\Available Bytes"
-            },
-            "threshold": 4000,
-            "windowSize": "PT5M",
-            "timeAggregation": "Average"
-          },
-          "actions": [
-            {
-              "$type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.RuleWebhookAction, Microsoft.WindowsAzure.Management.Mon.Client",
-              "odata.type": "Microsoft.Azure.Management.Insights.Models.RuleWebhookAction",
-              "serviceUri": "https://localhost/foo"
-            }
-          ]
-        }
-      }
-    }
-
-  ]
+{
```

Comment:
Is anything changed in this test file?

Reply:
![image](https://user-images.githubusercontent.com/38542602/62796638-8fab4780-ba8e-11e9-9d45-299c9d199b46.png)
Nope. I'm assuming VSCode did some auto-formatting on it.